core/execution/future/market.go

```diff
@@ -997,6 +997,67 @@ func (m *Market) parkAllPeggedOrders(ctx context.Context) []*types.Order {
 	return parked
 }
 
+func (m *Market) uncrossOrderAtAuctionEnd(ctx context.Context) {
+	if !m.as.InAuction() {
```

Comment:
As per the spec update, we also don't want to uncross if we are in an opening auction.